src/main/java/microsoft/exchange/webservices/data/core/service/ServiceObjectInfo.java

```diff
@@ -187,87 +175,76 @@ public Object createServiceObjectWithAttachmentParam(
-            return new MeetingCancellation(srv);
-          }
-        }, new ICreateServiceObjectWithAttachmentParam() {
-          public Object createServiceObjectWithAttachmentParam(
-              ItemAttachment itemAttachment, boolean isNew)
+    this.addServiceObjectType(XmlElementNames.MeetingCancellation, MeetingCancellation.class,
+                              new ICreateServiceObjectWithServiceParam() {
+                                public Object createServiceObjectWithServiceParam(ExchangeService srv)
+                                    throws Exception {
+                                  return new MeetingCancellation(srv);
+                                }
+                              }, new ICreateServiceObjectWithAttachmentParam() {
+          public Object createServiceObjectWithAttachmentParam(ItemAttachment itemAttachment, boolean isNew)
               throws Exception {
             return new MeetingCancellation(itemAttachment);
           }
         });
 
     // MeetingMessage
-    this.addServiceObjectType(XmlElementNames.MeetingMessage,
-        MeetingMessage.class,
-        new ICreateServiceObjectWithServiceParam() {
-          public Object createServiceObjectWithServiceParam(
-              ExchangeService srv) throws Exception {
-            return new MeetingMessage(srv);
-          }
-        }, new ICreateServiceObjectWithAttachmentParam() {
-          public Object createServiceObjectWithAttachmentParam(
-              ItemAttachment itemAttachment, boolean isNew)
+    this.addServiceObjectType(XmlElementNames.MeetingMessage, MeetingMessage.class,
+                              new ICreateServiceObjectWithServiceParam() {
+                                public Object createServiceObjectWithServiceParam(ExchangeService srv)
+                                    throws Exception {
+                                  return new MeetingMessage(srv);
+                                }
+                              }, new ICreateServiceObjectWithAttachmentParam() {
+          public Object createServiceObjectWithAttachmentParam(ItemAttachment itemAttachment, boolean isNew)
               throws Exception {
             return new MeetingMessage(itemAttachment);
           }
         });
 
     // MeetingRequest
-    this.addServiceObjectType(XmlElementNames.MeetingRequest,
-        MeetingRequest.class,
-        new ICreateServiceObjectWithServiceParam() {
-          public Object createServiceObjectWithServiceParam(
-              ExchangeService srv) throws Exception {
-            return new MeetingRequest(srv);
-          }
-        }, new ICreateServiceObjectWithAttachmentParam() {
-          public Object createServiceObjectWithAttachmentParam(
-              ItemAttachment itemAttachment, boolean isNew)
+    this.addServiceObjectType(XmlElementNames.MeetingRequest, MeetingRequest.class,
+                              new ICreateServiceObjectWithServiceParam() {
```

Comment:
I don't think we should align arguments, but I should check the Google guide for it.